### PR TITLE
Added models for invitiations, membership_roles, memberships, roles and subscriptions.

### PR DESCRIPTION
**Pull request by yinho999**

### CODE REVIEW

**Comment by yinho999** [comment]
image

File: migration/src/m20240322_202605_memberships.rs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        manager
+            .create_table(
+                table_auto(Memberships::Table)
+                    .col(pk_auto(Memberships::Id))
+                    .col(integer(Memberships::UsersId))
+                    .col(integer(Memberships::TeamsId))
+                    .foreign_key(
+                        ForeignKey::create()
+                            .name("fk-memberships-users")
+                            .from(Memberships::Table, Memberships::UsersId)
```
Comment (arferreira) [inline migration/src/m20240322_202605_memberships.rs:18]: What do you think about adding the pid as uuid in all entities, I don't think it's very interesting to expose a numeric id in cases where we will use it in the urls, what do you think about that?

**Review by arferreira — APPROVED** [review]
I left just a comment about the pid for external identifiers, overall lgtm

File: migration/src/m20240322_202605_memberships.rs (hunk 1 above)
Comment (yinho999) [reply migration/src/m20240322_202605_memberships.rs]: Agree, added pid for subscriptions table and members table

**Pull request merged**